### PR TITLE
[expo-camera]: Disable camera shutter sound via CameraPictureOptions

### PR DESCRIPTION
**Pull request by yuna5566**
# Why
We need a way to disable the camera shutter sound by adding a `shutterSound` key to `CameraPictureOptions`.

# How
Currently, disabling shutter sound relies on the volume of the device. Sometimes, we don't have to keep muting the device just to shut off the shutter sound. 

Also, on some Android devices, even muting does not remove the shutter sounds.

# Test Plan
Testing via device in `bare-expo`

### CODE REVIEW

File: packages/expo-camera/src/Camera.types.ts [hunk 1]
```diff
@@ -173,6 +173,11 @@ export type CameraPictureOptions = {
    * @hidden
    */
   maxDownsampling?: number;
+  /**
+   * To programatically disable the camera shutter sound
```
Comment (lukmccall) [inline packages/expo-camera/src/Camera.types.ts:177]:
```suggestion
   * To programmatically disable the camera shutter sound
```

**Review by lukmccall — APPROVED** [review]
Looking good!

**Comment by billhatwork** [comment]
This is looking great - just wondering if anyone knows whether this will be released soon? 🙇‍♂️

**Comment by fbegue-trx** [comment]
Looks awesome but not working on my side with the latest expo version (51.0.38) neither the latest expo-camera version (15.0.16)

**Comment by yuna5566** [comment]
> Looks awesome but not working on my side with the latest expo version (51.0.38) neither the latest expo-camera version (15.0.16) Testing my app in the expo app btw need to build to make it work?

It’s been merged, but it doesn’t seem to be published yet.

@lukmccall - it's been merged to main in a while. Any idea when will it be published?

**Comment by adback03** [comment]
Has anyone got this to work? I'm passing `shutterSound: false` to the `takePictureAsync` method, but I'm still hearing the shutter sound.

**Comment by atambo** [comment]
Is it possible to have this change backported to a 15.0.x version of expo-camera for expo 51? When upgrading from expo 50 to 51 I had to upgrade the expo-camera but now it makes a super loud shutter sound on android (the legacy camera did not make this sound).

**Comment by yuna5566** [comment]
@atambo - we are still in expo 51 and have this as `patch-package` for the time being.

**Pull request merged**